app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordWithPrescribedDrugs.kt

```diff
@@ -0,0 +1,26 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import androidx.room.Dao
+import androidx.room.Embedded
+import androidx.room.Query
+import androidx.room.Relation
+
+data class TeleconsultRecordWithPrescribedDrugs(
+    @Embedded
+    val teleconsultRecord: TeleconsultRecord,
+
+    @Relation(
+        parentColumn = "id",
+        entity = TeleconsultRecordPrescribedDrug::class,
+        entityColumn = "teleconsultRecordId"
+    )
+    val prescribedDrugs: List<TeleconsultRecordPrescribedDrug>
+
+) {
+  @Dao
+  interface RoomDao {
+
+    @Query("SELECT * FROM TeleconsultRecord")
+    fun getPrescribedUuidForTeleconsultRecordUuid(): List<TeleconsultRecordWithPrescribedDrugs>
```

Comment:
The name of this method implies that we should be passing in a teleconsult record ID and get the list of prescribed drug UUIDs which have been created as part of that teleconsult record but the method signature and the SQL query do not match this expectation.